setup.py

```diff
@@ -38,13 +38,13 @@ def get_package_data(name, extlist):
 
 
 # Requirements
-REQUIREMENTS = ['line_profiler', 'spyder>=4']
+REQUIREMENTS = ['line_profiler', 'spyder>=5']
```

Comment:
Not sure if the constraint should be `>=5.2` instead of `>=5`

Reply:
that's true, it should be 5.2